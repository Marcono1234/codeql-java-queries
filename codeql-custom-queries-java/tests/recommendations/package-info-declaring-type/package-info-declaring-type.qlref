./recommendations/package-info-declaring-type.ql
