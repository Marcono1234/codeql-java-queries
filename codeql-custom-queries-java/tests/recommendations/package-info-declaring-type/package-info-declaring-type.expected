| package-info.java:3:7:3:10 | Test | Type is declared in package-info.java |
